File: mambo-protocol/src/main/scala/org/mambo/protocol/enums/ExchangeErrorEnum.scala




















// Generated on 05/02/2013 20:27:17
// Scala version by Blackrush
package org.mambo.protocol.enums

import org.mambo.protocol.Enumeration

object ExchangeErrorEnum extends Enumeration {
  val requestImpossible = Value(1, "requestImpossible")
  val requestCharacterOccupied = Value(2, "requestCharacterOccupied")
  val requestCharacterJobNotEquiped = Value(3, "requestCharacterJobNotEquiped")
  val requestCharacterToolTooFar = Value(4, "requestCharacterToolTooFar")
  val requestCharacterOverloaded = Value(5, "requestCharacterOverloaded")
  val requestCharacterNotSuscriber = Value(6, "requestCharacterNotSuscriber")
  val requestCharacterRestricted = Value(7, "requestCharacterRestricted")
  val buyError = Value(8, "buyError")
  val sellError = Value(9, "sellError")
  val mountPaddockError = Value(10, "mountPaddockError")
  val bidSearchError = Value(11, "bidSearchError")
}

File: mambo-protocol/src/main/java/org/mambo/protocol/enums/ExchangeErrorEnum.java


// Generated on 05/08/2013 19:04:44
package org.mambo.protocol.enums;

public enum ExchangeErrorEnum {
    REQUEST_IMPOSSIBLE(1),
    REQUEST_CHARACTER_OCCUPIED(2),
    REQUEST_CHARACTER_JOB_NOT_EQUIPED(3),
    REQUEST_CHARACTER_TOOL_TOO_FAR(4),
    REQUEST_CHARACTER_OVERLOADED(5),
    REQUEST_CHARACTER_NOT_SUSCRIBER(6),
    REQUEST_CHARACTER_RESTRICTED(7),
    BUY_ERROR(8),
    SELL_ERROR(9),
    MOUNT_PADDOCK_ERROR(10),
    BID_SEARCH_ERROR(11);
    
    private int value;
    ExchangeErrorEnum(int value) { this.value = value; }
    public int value() { return value; }
    
    public static ExchangeErrorEnum valueOf(int value) {
    	for (ExchangeErrorEnum a : values()) {
    		if (a.value == value)
    			return a;
    	}
    	return null;
    }
    
}
